graphrag/index/api.py

```diff
@@ -50,11 +50,7 @@ async def build_index(
     list[PipelineRunResult]
         The list of pipeline run results
     """
-    try:
-        resolve_timestamp_path(config.storage.base_dir, run_id)
-        resume = True
-    except ValueError as _:
-        resume = False
+    resume = Path(config.storage.base_dir).exists()
```

Comment:
This means substitutions were already applied over the run_id right?

Reply:
Correct. At this point the timestamp paths have already been resolved with the run_id.